### DISCUSSION TITLE
Create a windows installer

### DISCUSSION THREAD

**Discussion by zrfisaac** [godotengine/godot-proposals#4313] [Platforms]
When I download Godot I always think why such a wonderful program like Godot does not have a windows installer?

I know very well and know how to use a open source script tool to create windows installer called [NSIS](https://nsis.sourceforge.io/Download)

My Idea is to create a friendly Godot installer

What do you guys think ?

**Comment by Mickeon** [comment]
How to install Godot (as of currently):

1. Download the [Godot executable from the official website](https://godotengine.org/download).
2. Done. Once it's extracted it can be run right out of the box. I don't believe it can be any simpler than this.

Alternatively, [it's on Steam too](https://store.steampowered.com/app/404790/Godot_Engine/). No worrying about installing, or manually updating to the latest version this way.

**Comment by Calinou** [comment]
There's already a proposal tracking this: https://github.com/godotengine/godot-proposals/issues/1432

See my comment at the end of the proposal for an update.

**Reply by zrfisaac** [reply]
Thanks I will have a look! 🙌

**Comment by ghost** [comment]
I've already made one... IN GODOT!

**Comment by anderlli0053** [comment]
Also... There is also Inno Setup ( https://jrsoftware.org/isinfo.php )....  the rival of NSIS, which is mostly script-based...

**Comment by fire-forge** [comment]
Why does Godot need an installer? What would the installer do differently than what we have now? I've never had any problems downloading a ZIP file from the Godot website or GitHub, extracting, and running the exe.

**Reply by Calinou** [reply]
The proposal I linked gives some reasons as to why an installer can be useful over a ZIP archive: https://github.com/godotengine/godot-proposals/issues/1432

Installers provide better system integration, especially for people less knowledgeable with doing this kind of stuff manually (or using Scoop).
